src/Illuminate/Auth/Console/AuthMakeCommand.php

```diff
@@ -80,6 +97,12 @@ protected function createDirectories()
         if (! is_dir(resource_path('views/auth/passwords'))) {
             mkdir(resource_path('views/auth/passwords'), 0755, true);
         }
+
+        if (! $this->option('no-tests')) {
```

Comment:
`if (! $this->option('no-tests') && ! is_dir(base_path('tests/Feature/Auth'))) {` is enough here